ros2doctor/ros2doctor/command/doctor.py

```diff
@@ -23,21 +24,31 @@ class DoctorCommand(CommandExtension):
     def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--report', '-r', action='store_true',
-            help='Print out doctor report.'
+            help='Print out all report info.'
+        )
+        parser.add_argument(
+            '--report_failed', '-rf', action='store_true',
```

Comment:
nit, I think `-` instead of `_` is more consistent with options on other CLI tools, so `--report-failed`.